src/signals/incident/components/form/MapSelectors/Caterpillar/Summary/__tests__/Summary.test.tsx

```diff
@@ -18,7 +18,6 @@ const contextValue: SelectValue = {
   ],
   meta: {
     icons: [],
-    legendItems: [],
```

Comment:
The file `src/signals/incident/components/form/MapSelectors/Caterpillar/Summary/__tests__/Summary.test.tsx` doesn't seem to be used any more. Can it be removed?

Reply:
the summary folder can indeed be removed